doc/source/getting_started/introduction.ipynb

```diff
@@ -0,0 +1,496 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# buffering the active geometry by 10 000 feet (geometry is already in feet)\n",
+    "gdf[\"buffered\"] = gdf.buffer(10000)\n",
+    "\n",
+    "# buffering the centroid geometry by 10 000 feet (geometry is already in feet)\n",
+    "gdf[\"buffered_centroid\"] = gdf[\"centroid\"].buffer(10000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = gdf[\"buffered\"].plot(alpha=.5)  # saving the first plot as an axis and setting alpha (transparency) to 0.5\n",
+    "gdf[\"buffered_centroid\"].plot(ax=ax, color=\"red\", alpha=.5)  # passing the first plot as an axis to the second\n",
+    "gdf[\"boundary\"].plot(ax=ax, color=\"white\", linewidth=.5)  # passing the first plot and setting linewitdth to 0.5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "User Guide\n",
+    "    \n",
+    "See more on [geometry creation and manipulation in the User Guide](../docs/user_guide/geometric_manipulations.rst).\n",
+    "</div>\n",
+    "\n",
+    "## Geometry relations\n",
+    "\n",
+    "We can also ask about the relations of different geometries. Using the geometries above, we can check which of the buffered boroughs intersect the original geometry of Brooklyn, i.e., is within 10 000 feet from Brooklyn.\n",
+    "\n",
+    "First, we get a polygon of Brooklyn."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "brooklyn = gdf.loc[\"Brooklyn\", \"geometry\"]\n",
+    "brooklyn"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "type(brooklyn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we can check which of the geometries in `gdf[\"buffered\"]` intersects it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf[\"buffered\"].intersects(brooklyn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Only Bronx (on the north) is more than 10 000 feet away from Brooklyn. All the others are closer and intersect our polygon.\n",
+    "\n",
+    "Alternatively, we can check which buffered centroids are entirely within the original boroughs polygons. In this case, both `GeoSeries` are aligned, and the check is performed for each row."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf[\"within\"] = gdf[\"buffered_centroid\"].within(gdf)\n",
+    "gdf[\"within\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can plot the results on the map to confirm the finding."
```

Comment:
Something that I find very typical to do is to use the boolean output of a relation to filter the data. So maybe we could also show that? (or eg only plot the geometries within instead of using the color to illustrate)

Reply:
Here I want to plot both, True and False to illustrate the spatial relation we just tested on the map. I'd leave it to plotting guide.

Reply:
It doesn't necessarily need to be included in this plotting example, but I  still think it is useful to include an example of filtering (boolean indexing) somewhere on the intro page, as in my experience that's a really typical use case.